packages/cursorless-vscode-e2e/src/suite/fixtures/scopes/haskell/argument.actual.iteration1.scope

```diff
@@ -0,0 +1,46 @@
+fib :: Integer -> Integer
+fib 0 = 0
+fib 1 = 1
+fib n = fib (n-1) + fib (n-2)
+
+---
+
+[#1 Range] =
+[#1 Domain] = 1:0-1:9
+0| fib :: Integer -> Integer
+
+1| fib 0 = 0
+  >---------<
```

Comment:
This makes sense as the domain, but I would think the range would be just that first `0`